sys-boot/zfsbootmenu/zfsbootmenu-3.0.1.ebuild

```
@@ -0,0 +1,53 @@
+# Copyright 2023-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit prefix
+
+DESCRIPTION="ZFS bootloader for root-on-ZFS systems"
+HOMEPAGE="https://zfsbootmenu.org"
+SRC_URI="https://github.com/zbm-dev/zfsbootmenu/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-stub-location.patch
+)
+
+RDEPEND="
+app-shells/fzf
+dev-lang/perl
+dev-perl/boolean
+dev-perl/Config-IniFiles
+dev-perl/Sort-Versions
+dev-perl/YAML-PP
+sys-apps/kexec-tools
+sys-block/mbuffer
+sys-fs/zfs
+sys-kernel/dracut
+"
+
+src_prepare() {
+	default
+	hprefixify bin/*
+	sed -e "s,#!/bin/sh,#!${BROOT}/bin/sh," \
+		-i install-tree.sh releng/version.sh
```

My suggestion to respect `BROOT` wasn't entirely correct, as we get a QA-warning about a useless sed when `${BROOT}` is empty. Sorry.

The following snippet also adds a missing `|| die`.

```suggestion
	if [[ -n ${BROOT} ]]; then
		sed -e "s,#!/bin/sh,#!${BROOT}/bin/sh," \
			-i install-tree.sh releng/version.sh || die
	fi
```

---

Thank you, applied